recipe-server/client/control/components/common/EnrollmentStatus.js

```diff
@@ -0,0 +1,83 @@
+import { Icon } from 'antd';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Link } from 'redux-little-router';
+import { connect } from 'react-redux';
+
+@connect((state, { record }) => {
```

Comment:
This isn't actually using `state` anywhere, as far as I can see. We probably don't want to `@connect` things that don't read from Redux. I would expect things like `isEnabled` and `isPaused` to be calculated in `render`, since they are derived from this component's props.

Reply:
That's actually how it started out, but for testing purposes I felt it was more straightforward to expose the `isEnabled`/`isPaused` props. I can disconnect it.

Reply:
I think using `isEnabled`/`isPaused` directly in tests skips out on testing an important bit of logic, that is, how those fields are determined.